teacher.rb

```diff
@@ -2,13 +2,15 @@
 
 class Teacher < Person
   attr_accessor :specialization
+  attr_reader :json_class
```

Comment:
- Hey, :wave: why add an attr_reader for the teacher class when you don't need it? I think this is redundant as you don't need the extra lines. Kindly remove it and also from your constructor as you've added another parameter in the teacher constructor to avoid unforeseen errors when you're working on adding the basic UI for the app.